app/controllers/idv/otp_verification_controller.rb

```diff
@@ -0,0 +1,113 @@
+module Idv
+  class OtpVerificationController < ApplicationController
+    include IdvSession
+    include TwoFactorAuthenticatable
+
+    # Skip TwoFactorAuthenticatable before action that depends on MFA contexts
+    # to redirect to account url for signed in users. This controller does not
+    # use or maintain MFA contexts
+    skip_before_action :check_already_authenticated
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_step_needed
+    before_action :handle_locked_out_user
+    before_action :confirm_otp_delivery_preference_selected
+    before_action :confirm_otp_sent, only: %i[show update]
+    before_action :set_code, only: %i[show update]
+    before_action :set_otp_verification_presenter, only: %i[show update]
+
+    def new
+      result = send_phone_confirmation_otp_form.submit
+      analytics.track_event(Analytics::IDV_PHONE_CONFIRMATION_OTP_SENT, result.to_h)
+      if result.success?
+        redirect_to idv_otp_verification_url
+      elsif send_phone_confirmation_otp_form.user_locked_out?
+        handle_too_many_otp_sends
+      else
+        redirect_to idv_otp_delivery_method_url
+      end
+    end
```

Comment:
We will also need to rescue any Twilio errors, like in the main 2FA controller in case the user enters a phone Twilio doesn't like, and so we can display an appropriate message.

Reply:
Hmm, interesting. In the main 2FA controllers, the user is shown the errors on the phone entry screen. Here is would appear on the OTP delivery method selection screen. I wonder what the best way to deal with that is...